app/models/miq_group.rb

```diff
@@ -94,9 +94,7 @@ def self.seed
   end
 
   def self.get_ldap_groups_by_user(user, bind_dn, bind_pwd)
-    auth = VMDB::Config.new("vmdb").config[:authentication]
-    auth[:group_memberships_max_depth] ||= User::DEFAULT_GROUP_MEMBERSHIPS_MAX_DEPTH
-
+    max_depth = ::Settings.authentication.group_memberships_max_depth ||= User::DEFAULT_GROUP_MEMBERSHIPS_MAX_DEPTH
```

Comment:
be careful of the `||=`.  We should not be modifying the global settings directly.